ocfweb/lab_reservations/templates/lab_reservations/index.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+
+            <form method="post" action="{% url 'request_reservation' %}">
+                {% csrf_token %}
+                <p>
+                    This form is <strong>only</strong> for student groups desiring to
+                    use the OCF's lab space. If you want to create an individual account
+                    please visit <a href="{% url 'register' %}">this link</a>. If you
+                    want to request an account for a student group, please visit
+                    a staff member in the OCF lab during
+                    <a href="{% url 'staff-hours' %}">staff hours</a>.
+                </p>
+
+                <hr />
+
+                <p>{{form.real_name|bootstrap}}</p>
+
+                <hr />
+                Please provide a UC Berkeley e-mail address. We will use this
+                to send you a confirmation email in addition to any necessary
+                follow-ups (including an approval email).
+                <p>{{form.contact_email|bootstrap}}</p>
+                <p>{{form.verify_contact_email|bootstrap}}</p>
+
+                <hr />
+
+                <p>{{form.student_group|bootstrap}}</p>
+                <p>{{form.reason|bootstrap}}</p>
+
+                <hr />
+
+                <p>{{form.date|bootstrap}}</p>
+                <p>{{form.starttime|bootstrap}}</p>
+                <p>{{form.endtime|bootstrap}}</p>
+
+
+                <div class="well">
+                    <p><strong>
+                        Please read our lab reservation policy
+                        <a href=https://www.ocf.berkeley.edu/docs/services/lab/lab-reservation-policy/>
+                         here</a>; you are responsible for
+                        knowing and abiding by it.
+                    </strong></p>
+
+                    <p><strong>
+                        I have read, understood, and agreed to follow the policy linked above.
+                         In addition, I agree not to hold the University
+                        of California, the Associated Students of the University of
+                        California, or the Open Computing Facility (OCF)
+                        responsible for lost files or other misfortunes which may
+                        result from my use of the OCF's facilities.
+
+                        Additionally, I certify that I am eligible for an OCF
+                        account according to the
+                        <a href="{% url 'doc' 'membership/eligibility' %}">OCF account eligibility policy</a>.
```

Comment:
why not just require a group account if that's the idea?

this entire template is pretty confusingly written

Reply:
Apparently, we want to have this page open to the public and not behind a group account login, because groups without OCF group accounts should be allowed to reserve the lab. I don't think making a group account is a signficant hurdle for the kinds of groups we'd approve of using the lab, however.

Reply:
I do think we should allow other groups to use the lab, for instance I'm not sure the FAFSA people we let use the lab had a group account associated with them, yet they were a great example of a group that should be using this.